src/components/core/Sidenav.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faDashboard, faBriefcase, faUser, faUsers, faCogs, faLegal } from '@fortawesome/freesolid-svg-icons';
+
+class Sidenav extends Component {
+    render() {
+        const FaDashboard = <FontAwesome icon={["fal", "dashboard"]} />,
+            FaBriefcase = <FontAwesome icon={["fal", "briefcase"]} /> ,
+            FaUser = <FontAwesome icon={["fal", "user"]} />,
+            FaUsers = <FontAwesome icon={["fal", "users"]} />,
+            FaCogs = <FontAwesome icon={["fal", "cogs"]} />,
+            FaLegal = <FontAwesome icon={["fal", "legal"]} />;
```

Comment:
There seems to be a problem with how you have declared the Fontawesome and how you use it, it is preventing the app from from running please check it out.